examples/angular/.bazelrc

```diff
@@ -1,4 +1,8 @@
 import %workspace%/../../common.bazelrc
 # Use the Angular IVy compiler
 # See https://github.com/angular/angular/blob/master/docs/BAZEL.md#various-flags-used-for-tests
-build --define=compile=aot
\ No newline at end of file
+build --define=compile=aot
+
+# Override the one in common
```

Comment:
why override it instead of removing it?

Reply:
There's a comment in the common saying it's needed for `pkg_tar`, is this no longer true?

Reply:
ah wait, I was reading the wrong .bazelrc, not common